diff --git a/node_modules/@ethersproject/experimental/lib/nonce-manager.d.ts b/node_modules/@ethersproject/experimental/lib/nonce-manager.d.ts
index e981f31..926f05c 100644
--- a/node_modules/@ethersproject/experimental/lib/nonce-manager.d.ts
+++ b/node_modules/@ethersproject/experimental/lib/nonce-manager.d.ts
@@ -4,7 +4,6 @@ export declare class NonceManager extends ethers.Signer {
     _initialPromise: Promise<number>;
     _deltaCount: number;
     constructor(signer: ethers.Signer);
-    get provider(): ethers.providers.Provider;
     connect(provider: ethers.providers.Provider): NonceManager;
     getAddress(): Promise<string>;
     getTransactionCount(blockTag?: ethers.providers.BlockTag): Promise<number>;
