diff --git a/node_modules/@types/ember-data/index.d.ts b/node_modules/@types/ember-data/index.d.ts
index 78a645d..2b49527 100644
--- a/node_modules/@types/ember-data/index.d.ts
+++ b/node_modules/@types/ember-data/index.d.ts
@@ -964,7 +964,7 @@ export namespace DS {
          */
         snapshots(): Snapshot[];
     }
-    class Snapshot<K extends keyof ModelRegistry = any> {
+    class Snapshot<K extends keyof ModelRegistry = keyof ModelRegistry> {
         /**
          * The underlying record for this snapshot. Can be used to access methods and
          * properties defined on the record.
