diff --git a/node_modules/emberfire/adapters/realtime-database.d.ts b/node_modules/emberfire/adapters/realtime-database.d.ts
index 0ae4d46..1a93612 100644
--- a/node_modules/emberfire/adapters/realtime-database.d.ts
+++ b/node_modules/emberfire/adapters/realtime-database.d.ts
@@ -7,13 +7,13 @@ import { database } from 'firebase/app';
 declare const RealtimeDatabaseAdapter_base: Readonly<typeof DS.Adapter> & (new (properties?: object | undefined) => {
     namespace: string | undefined;
     firebaseApp: import("@ember/object/computed").default<FirebaseAppService, FirebaseAppService>;
-    databaseURL: undefined;
+    databaseURL?: string;
     database: RSVP.Promise<database.Database> | undefined;
     defaultSerializer: string;
 } & DS.Adapter) & (new (...args: any[]) => {
     namespace: string | undefined;
     firebaseApp: import("@ember/object/computed").default<FirebaseAppService, FirebaseAppService>;
-    databaseURL: undefined;
+    databaseURL?: string;
     database: RSVP.Promise<database.Database> | undefined;
     defaultSerializer: string;
 } & DS.Adapter);
diff --git a/node_modules/emberfire/services/firebase-app.d.ts b/node_modules/emberfire/services/firebase-app.d.ts
index ad1eed7..ddf1906 100644
--- a/node_modules/emberfire/services/firebase-app.d.ts
+++ b/node_modules/emberfire/services/firebase-app.d.ts
@@ -3,10 +3,10 @@ import Service from '@ember/service';
 import Ember from 'ember';
 import FirebaseService from './firebase';
 declare const FirebaseAppService_base: Readonly<typeof Service> & (new (properties?: object | undefined) => {
-    name: undefined;
+    name?: string;
     firebase: import("@ember/object/computed").default<FirebaseService, FirebaseService>;
 } & Service) & (new (...args: any[]) => {
-    name: undefined;
+    name?: string;
     firebase: import("@ember/object/computed").default<FirebaseService, FirebaseService>;
 } & Service);
 export default class FirebaseAppService extends FirebaseAppService_base {
